### PR TITLE
My WIP TMO50 layout

### PR DESCRIPTION
**Pull request by PyrooL**
## Types of changes

- [ ] Core
- [ ] Bugfix
- [ ] New Feature
- [ ] Enhancement/Optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/Layout/Userspace (addition or update)
- [ ] Documentation

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document. (https://docs.qmk.fm/#/contributing)
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage). **Does using my keymap on a daily basis count?**

### CODE REVIEW

**Comment by PyrooL** [comment]
A lot of the commits are just random maintenance on my end.

File: keyboards/tmo50/keymaps/default/keymap.c [hunk 1]
```diff
@@ -43,6 +43,7 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
   ),
 
   // Fn3 layer
+<<<<<<< HEAD
```
Comment (drashna) [inline keyboards/tmo50/keymaps/default/keymap.c]:
```suggestion
```

**Comment by drashna** [comment]
And no problem, on the commit history.  It will be squashed when merged. :)

**Comment by PyrooL** [comment]
lol whoops thought i fixed the conflicts, not sure how that happened.

**Comment by drashna** [comment]
We merged stuff that caused merge conflicts for a lot of boards, sorry. :(

**Comment by PyrooL** [comment]
Any update?

**Comment by drashna** [comment]
Thanks!

**Pull request merged**